daffodil-sapi/src/main/scala/org/apache/daffodil/sapi/packageprivate/Utils.scala

```diff
@@ -80,6 +82,8 @@ private[sapi] object ValidationConversions {
       case SValidationMode.Off => ValidationMode.Off
       case SValidationMode.Limited => ValidationMode.Limited
       case SValidationMode.Full => ValidationMode.Full
+      // todo;; hacking this out for now
+      case _ => ValidationMode.Off
```

Comment:
Not sure how to approach this

Reply:
modeFromScala is not used. I think we can delete this.